python/FCCAnalysisRun.py

```diff
@@ -301,14 +301,24 @@ def sendToBatch(rdfModule, chunkList, process, analysisFile):
         LOGGER.warning('I/O error(%i): %s', e.errno, e.strerror)
         time.sleep(10)
         frun_condor = open(frunfull_condor, 'w')
+    makeConfig = open(localDir+'/build/CMakeFiles/CMakeConfigureLog.yaml', 'r')
```

Comment:
Can `with` context be used here?

Reply:
Also, what should happen if the file is not found? The name of the build directory is not guaranteed to be `build`.

Reply:
Thanks, Juraj for the good points. 
I did not know the `/build` directory could be missing. Is the build still successful in that case? Maybe you have a better way to check the build environment than to look at the `CMakeConfigureLog.yaml`?

For the moment, I updated the script now that if the file is not found, a warning is printed, the `OpSysAndVer` is not specified in the `job_desc`, and the jobs are submitted to default "AlmaLinux9" machines.